app/models/source.js

```diff
@@ -5,7 +5,9 @@
 
 var modelFactory = require('app/factories/model');
 var nameMethods = require('./methods/name');
+// a way to interporlate strings to get a final URL
 var templateCompiler = require('es6-template-strings');
+var SourceContentType = require('app/models/sourceContentType');
```

Comment:
Please order required modules alphabetically at the top of the file as possible and integrate this rule to enforce: https://eslint.org/docs/rules/sort-imports